OpenXMLSDK.Engine/ReportEngine/DataContext/SubstitutableStringModel.cs

```diff
@@ -50,10 +46,15 @@ public SubstitutableStringModel(string renderPattern, ContextModel dataSource) :
         /// <returns></returns>
         public string Render(IFormatProvider formatProvider)
         {
-            var renders = new List<string>();
+            if (DataSource is null || DataSource.Data is null || DataSource.Data.Values is null)
+                return string.Empty;
 
+            var renders = new List<string>();
             foreach (var baseModel in DataSource.Data.Values)
             {
+                if (baseModel is null)
```

Comment:
Please remove this continue statement by a LinqQuery (.Where(e => e != null) in the foreach

Reply:
Ok, change.